Socials.yml

```diff
@@ -127,6 +127,7 @@ LiveJournal:
   - livejournal.com
 
 Mastodon:
+  - org.joinmastodon.android
```

Comment:
Would you mind adding that further down in the list? The first entry should represent the most common url

Reply:
I move it to the bottom of the Mastodon block. Is it ok?